app-backup/borgbackup/borgbackup-1.2.6.recipe

```diff
@@ -11,29 +11,29 @@ HOMEPAGE="https://www.borgbackup.org/"
 COPYRIGHT="2015-2023 The Borg Collective
 	2010-2014 Jonas Borgström"
 LICENSE="BSD (3-clause)"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="https://github.com/borgbackup/borg/archive/refs/tags/$portVersion.tar.gz"
-CHECKSUM_SHA256="fd9732b7c71e4f3d12b67e54d2fcda18e1c205c55a93df21ff80d006ab047214"
+CHECKSUM_SHA256="a5af10fab99a135638cf08ab58d14b4ed998cd93c68b76e94d8183ef2c529499"
 SOURCE_FILENAME="borgbackup-$portVersion.tar.gz"
 SOURCE_DIR="borg-$portVersion"
-SOURCE_URI_2="https://github.com/hynek/argon2-cffi/archive/refs/tags/21.3.0.tar.gz"
-CHECKSUM_SHA256_2="1f5cdc94200eccdf8c63c1cc5c0c72e38076d8e81968218b0ec0573fd72b2ba4"
-SOURCE_FILENAME_2="argon2-cffi-21.3.0.tar.gz"
 
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
+pythonVersion="3.10"
+pythonPackage="python310"
+
 PROVIDES="
 	$portName = $portVersion
 	cmd:borg
 	cmd:borgfs
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	argon2_cffi_bindings${secondaryArchSuffix}_python39
-	cffi${secondaryArchSuffix}_python39
-	msgpack_python39
-	packaging_python39
+	argon2_cffi_bindings${secondaryArchSuffix}_$pythonPackage
```

Comment:
@OscarL I'm wondering if we can use **argon2_cffi_bindings_$pythonPackage** here? (on 32bit, haven't checked there yet) :)